dialogy/plugins/text/list_search_plugin/__init__.py

```diff
@@ -204,7 +204,7 @@ def search_regex(
                     final_match = match_value
                     match_span = result.span()
         if final_match:
-            return (match_text, entity_type, final_match, match_span, float(0))
+            return (match_text, entity_type, final_match, match_span, float(1))
```

Comment:
Use 0.0, 1.0 instead of `float(0)`, `float(1)`.

Reply:
implementing this change